components/languages-section/LanguagesSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { CheckedTitle, Section, SectionItem } from '..';
+import { ComponentWithT } from '../../lib/types';
+import { Skill } from '../../lib/types/talent';
+import { sortSkill } from '../../lib/utils/arrays';
+import { SkillItem } from '../skill-item/SkillItem';
+
+export interface LanguageSectionProps extends ComponentWithT {
+  languages: Skill[];
+  handleEdit: () => void;
+}
+
+export const LanguageSection = ({
```

Comment:
also prefer export default

Reply:
Both fine. But since this will be a lot of rather boring but time consuming (we have to check that no imports get broken) maintenance work, I suggest to move this (plus absolute imports) to another pull request. Also we probably want to decide how we feel about index in components.